util/ctags.sh

```diff
@@ -11,9 +11,9 @@
 # Usage: ./util/ctags.sh [...arguments for ctags...]
 #
 # This script runs ctags twice. In the first pass, ctags extract macro
-# definitions. readtags that is part of Universal Ctags converts them
+# definitions. readtags that are part of Universal Ctags converts them
```

Comment:
I think this is wrong - `readtags` is a (singular) tool whose name happens to end with an `s`.

This would probably be better off parenthesised as `readtags (part of Universal Ctags) converts them...`

Reply:
If it's a tool, let's explicitly mention it. So that readability would improve.